x/ibc/core/02-client/types/params.go

```diff
@@ -0,0 +1,70 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var (
+	// DefaultAllowedClients are "Solo Machine" and "Tendermint"
+	DefaultAllowedClients = []string{"Solo Machine", "Tendermint"}
```

Comment:
Where are these strings from? Can we call the methods on the client types directly?

Reply:
dep cycles. I could move them to exported/ as well